plasma/root_chain/contracts/RootChain.sol

```diff
@@ -276,7 +276,10 @@ contract RootChain {
             // FIXME: handle ERC-20 transfer
             require(address(0) == _token, "Token must be ETH.");
 
-            currentExit.owner.transfer(currentExit.amount);
+            if (address(0) != currentExit.owner) {
```

Comment:
Tiny nitpick but for logical clarity I'd reverse these two: `currentExit.owner != address(0)`

Reply:
Okay